tensorflow_probability/python/bijectors/masked_autoregressive.py

```diff
@@ -25,7 +25,7 @@
 import tensorflow.compat.v1 as tf1
 import tensorflow.compat.v2 as tf
 
-from tensorflow_probability.python.bijectors import affine_scalar
+from tensorflow_probability.python.bijectors import shift, scale
```

Comment:
Note that shift is also a name used below, which is what is making Travis fail. 
maybe:
`from tensorflow_probability.python.bijectors import shift as shift_bijector`
Also please put a single import per line.

Reply:
Sorry for late reply. Seems like tests are still failing.